pkg/channel/attributes/knative_error.go

```diff
@@ -17,22 +17,26 @@ limitations under the License.
 package attributes
 
 import (
+	"net/url"
+
 	"github.com/cloudevents/sdk-go/v2/binding"
 	"github.com/cloudevents/sdk-go/v2/binding/transformer"
 )
 
 const (
+	KnativeErrorDestExtensionKey       = "knativeerrordest"
 	KnativeErrorCodeExtensionKey       = "knativeerrorcode"
 	KnativeErrorDataExtensionKey       = "knativeerrordata"
 	KnativeErrorDataExtensionMaxLength = 1024
 )
 
 // KnativeErrorTransformers returns Transformers which add the specified error code and data extensions.
```

Comment:
nit: update the doc ?

Reply:
oh - did you mean the code comment below?